resource/ckb-miner.toml

```diff
@@ -8,6 +8,7 @@
 data_dir = "data"
 
 [chain]
+tx_block_stat_enable = false
```

Comment:
You should better comment this option which let the config file to be backward compatible.

You don't have to add it explicitly because there is a default value and the default value is same as the value you want to set here.

See the follow example which add a new option `use_binary_version_as_message_prefix`:
https://github.com/nervosnetwork/ckb/blob/d7a330fde8800ac242a2ccfe195bfe7ea4d50791/resource/ckb.toml#L183-L184

p.s. Same suggestion for the `resource/ckb.toml`.